src/main/webapp/chat.js

```diff
@@ -19,10 +19,12 @@ async function addToChat() {
 }
 
 function updateChat(data) {
-  let newMessage = document.createElement("li");
-  newMessage.appendChild(document.createTextNode(data.message));
-  let chat = document.getElementById("chat");
-  chat.appendChild(newMessage);
+  let message = data.message;
+  let newChatItem = `<p class="message"><span class="username">User: </span>${message}</p>`;
```

Comment:
Is the plan to eventually replace "User" with the username?

Reply:
Yeah, should see that in next PR